man/gemfile.5.ronn

```diff
@@ -64,6 +64,18 @@ the Ruby version that the engine is compatible with.
 
     ruby "1.9.3"
 
+If your application is a library which needs to be tested on more than one
+Ruby version, you can tell Bundler to use the current Ruby version as the
+required one:
+
+    ruby RUBY_VERSION
```

Comment:
I guess it can mean "using this _will_ store this Ruby version requirement in the Gemfile.lock".